README.md

```diff
@@ -17,6 +17,7 @@ A curated list of awesome resources related to [Railway](https://railway.app/).
   - [Utilities](#utilities)
 - [Community](#community)
   - [Built on Railway](#built-on-railway)
+  - [GitHub Actions](#github-actions)
```

Comment:
This is a little too specific IMO. Can we change this to Utilities and make it more generic?